src/infra/monitoring/grafana/terraform/stamps/app-service.tf

```diff
@@ -6,6 +6,8 @@ resource "azurerm_service_plan" "asp" {
   os_type             = "Linux"
   sku_name            = "P1v2"
 
+  zone_balancing_enabled = true # Balance Service Plan across Availability Zones (AZs).
```

Comment:
this requires at least three app service instances, I believe. Not sure if thats worth it for grafana right now. maybe just add it and comment it out (with a comment why)?

Reply:
Yep, was thinking the same - especially as the current PostgreSQL instance doesn't support zones at all.

Reply:
```suggestion
  zone_balancing_enabled = false # Balance Service Plan across Availability Zones (AZs)
  # This is currently disabled as the database backend is not using AZs.
```